src/redux/books/books.js

```diff
@@ -0,0 +1,24 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
+export const addBook = (title, author) => ({
+  type: ADD_BOOK,
+  title,
+  author,
+});
+
+export const removeBOok = (title) => ({
+  type: REMOVE_BOOK,
+  title,
+});
+
+export const bookReducer = (state = [], action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...state, { title: action.title, author: action.author }];
+    case REMOVE_BOOK:
+      return state.filter((obj) => obj.title !== action.title);
```

Comment:
- [OPTIONAL] You did an amazing job but I highly recommend filtering by the book `id` which is the most efficient way to do this.

   `return state.filter(book => book.id !== id);`